src/frontend/js/components/AshleyEditor/index.tsx

```diff
@@ -26,7 +36,7 @@ import {
   BlockquoteButton,
   CodeBlockButton,
 } from '@draft-js-plugins/buttons';
-
+import { MentionData } from '@draft-js-plugins/mention';
```

Comment:
You could remove this line and import `MentionData` above with `createMentionPlugin` and `defaultSuggestionFilter`